habitat-baselines/habitat_baselines/common/construct_vector_env.py

```diff
@@ -93,7 +94,7 @@ def construct_envs(
         vector_env_cls = VectorEnv
 
     envs = vector_env_cls(
-        make_env_fn=make_gym_from_config,
+        make_env_fn=make_env_fn,
```

Comment:
Same as above comment. This will allow us to pass custom env wrapper [here](https://github.com/facebookresearch/home-robot/blob/4141a5f7093246a1861d12f31b621e6b8d755066/projects/habitat_ovmm/eval_vectorized.py#L72)